operator/duties/fork_listener.go

```diff
@@ -0,0 +1,48 @@
+package duties
```

Comment:
I don't think this is part of this package.

Reply:
deleted